Packs/AzureStorageTable/Integrations/AzureStorageTable/AzureStorageTable.yml

```
@@ -0,0 +1,278 @@
+    - default: false
+      description: Unique identifier for the partition within a given table
+      isArray: false
+      name: partition_key
+      required: true
+      secret: false
+    - default: false
+      description: Unique identifier for an entity within a given partition.
+      isArray: false
+      name: row_key
+      required: true
+      secret: false
+    - default: false
+      description: 'Entity records in JSON format: { "Key1": Value1, "Key2": Value2}'
+      isArray: false
+      name: entity_fields
+      required: true
+      secret: false
+    deprecated: false
+    description: Insert a new entity into a table.
+    execution: false
+    name: azure-storage-table-entity-insert
+    outputs:
+    - contextPath: AzureStorageTable.Entity.PartitionKey
+      description: Entity partition key.
+      type: String
+    - contextPath: AzureStorageTable.Entity.RowKey
+      description: Entity row key.
+      type: String
+    - contextPath: AzureStorageTable.Entity.Timestamp
+      description: Entity last update UTC time.
+      type: Date
+    - contextPath: AzureStorageTable.Entity.table_name
+      description: Table Name.
+      type: String
+  - arguments:
+    - default: false
+      description: Table name.
+      isArray: false
+      name: table_name
+      required: true
+      secret: false
+    - default: false
+      description: Unique identifier for the partition within a given table.
+      isArray: false
+      name: partition_key
+      required: true
+      secret: false
+    - default: false
+      description: Unique identifier for an entity within a given partition.
+      isArray: false
+      name: row_key
+      required: true
+      secret: false
+    - default: false
+      description: 'Entity records in JSON format: { "Key1": Value1, "Key2": Value2}'
+      isArray: false
+      name: entity_fields
+      required: true
+      secret: false
+    deprecated: false
+    description: Update an existing entity in a table.The Update Entity command does
+      not replace the existing entity.
+    execution: false
+    name: azure-storage-table-entity-update
+  - arguments:
+    - default: false
+      description: Table name.
+      isArray: false
+      name: table_name
+      required: true
+      secret: false
+    - default: false
+      description: Unique identifier for the partition within a given table.
+      isArray: false
+      name: partition_key
+      required: true
+      secret: false
+    - default: false
+      description: Unique identifier for an entity within a given partition.
+      isArray: false
+      name: row_key
+      required: true
+      secret: false
+    - default: false
+      description: 'Entity records in JSON format: { "Key1": Value1, "Key2": Value2}'
+      isArray: false
+      name: entity_fields
+      required: true
+      secret: false
+    deprecated: false
+    description: Replace an existing entity in a table.The Replace Entity command
```

small but important note
```suggestion
    description: Replace an existing entity in a table. The Replace Entity command
```